hardware/Comply-to-the-Regulations.md

```diff
@@ -0,0 +1,394 @@
+## 4.1. FCC requirements in Sub-GHz bands
+Silicon Labs' products can be mainly applied to FCC-Part-15.231 and FCC-Part-15.247 regulations for sub-G bands. It regulates 260 to 470MHz frequency band in Part-15.231, and 902 to 928MHz frequency band in Part-15.247.
+- Part-15.231
+  Typical applications are Door openers, Alarm systems, Remote Switches, and Control links in this regulation. There are severy typical frequency bands as 315MHz, 433MHz, and 470MHz.
+  - Tx power limits
+    The power limits are listed in field strength(dBuV/m) in 3 meters distance. The power is measured with average detector, and it means that the maxium power can be relaxed by using duty-cycle-factor(DCF).And it limits vary as a function of frequency: 3,750 uV/m at 260 MHz, to 12,500 uV/m at 470 MHz. And these field strength can be converted into dBm(in EIRP):
+    - -19.6 dBm at 315 MHz
+    - -14.4 dBm at 433 MHz
+    - -13.3 dBm at 470 MHz
+
+  - Tx bandwidth requirement
+
+    The 20dB emission bandwidth should be no wider than 0.25% of the center operation frequency.
+
+  - Spurs emission limits
+    General requirement is very loose in this part, the limit is -20dBc everywhere. But for the spurs lie in restrict bands defined in Part-15.205, the spurs including harmonics will be less than -49dBm for frequency < 960MHz, and be less than -41dBm for frequency >960MHz.
+    Since the fundamental power is very low, the harmonics limits can be achieved very easy by only adding a 3-order low pass filter in the RF path.
+
+  - Rx requirements
+    
+    No sensitivity requirement. But the limits for harmonics or spurs in a restricted band is defined in “FCC Part 15.109 Unintentional Radiators” as following:
+    - Spurious falls below 960 MHz = -49.2 dBm EIRP (max) (15.209.a);
+    - Spurious falls above 960 MHz = -41.2 dBm EIRP (max) (15.209.a).
+
+- Part-15.247
+  Typical applications are Weather station reporting, Automatic meter reading, and long range communication in this part. It covers wide bandwidth applications in 902~928MHz frequency band:
+  a)Frequency Hopping Spread Spectrum (FHSS);
+  b)Wideband digital modulation (BWMOD_6dB > 500 kHz);
+  c)Direct Sequence Spread Spectrum; 
+
+  - Tx power limits
+    Since it is a wide band appliction in this regulation. The maximum output power can be high since the band "spread out" effection. The fundamental output power should be a Peak measurement.
```

Comment:
Change "appliction" to "application".  Change "regulation.  The" to "regulation, the".  Change "effection" to "effectively".

Reply:
Corrected.